drivers/temperature/max31865pmb1/max31865.c

```diff
@@ -69,20 +71,29 @@ int max31865_init(struct max31865_dev **device,
 	if (!init_param)
 		return -EINVAL;
 
-	descriptor = (struct max31865_dev *)calloc(1, sizeof(*descriptor));
+	descriptor = (struct max31865_dev *)no_os_calloc(1, sizeof(*descriptor));
```

Comment:
the alloc stuff should go in separate commit.
Also, please follow the proper format for commit title (i.e. `drivers:temperature:max31865pmb: add enhancements`) and the 50/72 commit message rule.

Reply:
Does this mean that I need to have 3 commits: formatting corrections, enhancements, and no_os_alloc function usage?

Reply:
Yes